prometheus/README.md

```diff
@@ -0,0 +1,106 @@
+# Prometheus Scraping & Grafana Dashboard for Jitsi
+
+## Overview
+
+This project aims to integrate **Prometheus** and **Grafana** with Jitsi to monitor and visualize performance metrics. It enhances Jitsi’s operational efficiency by providing detailed insights into resource utilization, network traffic, and service availability.
+
+## Features
+
+- **Prometheus Integration**: Collects metrics from Jitsi containers.
+- **Grafana Dashboards**: Visualizes the metrics for easy analysis.
+
+## Installation
+
+### Prerequisites
+
+- Docker
+- Docker Compose
+
+### Steps
+
+1. **Clone the Repository**
+
+   ```bash
+   git clone https://github.com/yourusername/prometheus-scraper-jitsi-meet.git
+   cd prometheus-scraper-jitsi-meet
+   ```
+
+2. **Setup Jitsi with Docker Compose**
+
+   Follow the [Jitsi Docker](https://github.com/jitsi/docker-jitsi-meet) setup instructions. <br>
+   Also, you could follow Self - Hosting guide of Jitsi Meet: [Jitsi handbook](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker/)
+
+3. **Deploy Prometheus and Grafana**
+
+   Adding the service in different files which allows for better user control. These files are added at root level.<br>
+
+   Add Prometheus service to your `prometheus.yml` which turns up the container:
```

Comment:
Is this really what the user is supposed to do to enable prometheus? I thought that's already part of prometheus.yml and the user simply needs to add `-f prometheus.yml` to the docker-compose commain